cluster/examples/kubernetes/ceph/rgw-external.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   ports:
   - name: rgw
-    port: 80
+    port: 8080 #internal port used by the rgw daemon
     protocol: TCP
-    targetPort: 80
+    targetPort: 80 #service port mentioned in object store crd
```

Comment:
```suggestion
    targetPort: 80 # service port mentioned in object store crd
```

Reply:
That's the other way around `targetPort` is always the internal port where the `port` is the same as the CR setting.

Reply:
Interesting

Reply:
It is still wrong.